vendor/src/github.com/tonistiigi/fifo/fifo.go

```diff
@@ -1,14 +1,14 @@
 package fifo
 
 import (
-	"context"
 	"io"
 	"os"
 	"runtime"
 	"sync"
 	"syscall"
 
 	"github.com/pkg/errors"
+	"golang.org/x/net/context"
```

Comment:
`docker` still uses `golang.org/x/net/context` almost everywhere except in 3 files, so it should still work (also, this file wouldn't end up being imported into docker by `hack/vendor.sh`).

I'll give it a try just to be sure though.

Thanks for pointing it out